tests/transform/test_dynamodb_full.py

```diff
@@ -71,6 +72,7 @@ def test_to_sql_list_of_objects():
                         "utm_source": "google",
                     }
                 ]
-            }
+            },
+            "untyped": {},
```

Comment:
Shouldn't we add a test including `untyped` columns?

Reply:
You are right on the spot. Another test for that is yet missing. Thanks a stack!